spec/text-editor-spec.coffee

```diff
@@ -916,6 +916,12 @@ describe "TextEditor", ->
         editor.setCursorBufferPosition([3, 1])
         expect(editor.getCurrentParagraphBufferRange()).toBeUndefined()
 
+    describe "getCursorAtScreenPosition(screenPosition)", ->
+      it "returns the cursor at the given screenPosition", ->
+        cursor1 = editor.addCursorAtScreenPosition([0, 2])
+        cursor2 = editor.getCursorAtScreenPosition(cursor1.getScreenPosition())
+        expect(cursor2.marker).toBe cursor1.marker
```

Comment:
Just realized that the code that I commented on here was actually based on an existing test right below. So what you did was totally reasonable. I see that in a later commit, you actually updated that other test and not this one. Could you do the same thing here?